src/main/resources/META-INF/rewrite/assertj.yml

```
@@ -45,6 +45,233 @@ recipeList:
       fullyQualifiedTargetTypeName: "org.assertj.core.api.Assertions"
   - org.openrewrite.java.UseStaticImport:
       methodPattern: "org.assertj.core.api.Assertions *(..)"
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.testing.assertj.SimplifyChainedAssertJAssertions
+displayName: Simplify AssertJ chained assertions
+description: Replace AssertJ chained assertions with dedicated assertions that function the same.
+tags:
+  - testing
+  - assertj
+recipeList:
+  # Simplify Chained AssertJ Assertions
+  # String Assertions
+  - org.openrewrite.java.testing.assertj.SimplifyChainedAssertJAssertion:
+      chainedAssertion: isEmpty
```

I think it's likely we'll want to move to something like matching `isEmpty(String)` or `isEmpty(CharSequence)` hear to prevent those too eager matches on `isEmpty(Path)`

---

So you think all of the tests that use `isEmpty` should be moved up to the top of the String assertions? I want to make sure I am understanding this correctly.

---

I'd meant more something along the lines of what you arrived at [in this comment](https://github.com/openrewrite/rewrite-testing-frameworks/pull/385#issuecomment-1659360505), but then perhaps (if possible) as part of the same input, unless that leads to less than optimal string wrangling. So for instance for this line:
```suggestion
      chainedAssertion: isEmpty(String)
```